docs/source/template_options.rst

```diff
@@ -71,7 +71,7 @@ This will be:
 
     * Your GitHub username, if you're working outside of an organization. 
 
-This is used to construct URLs to your project, as in: ``https://github.com/{{project_organization}}/{{project_name}}``.
+This is used to construct URLs to your project, as in: ``git@github.com:{{project_organization}}/{{project_name}}``.
```

Comment:
This line in documentation is specifically about constructing web URLs, so should remain HTTP.